src/Diagnostics.RuntimeHost/Services/DiagnosticTranslator/DiagnosticTranslatorService.cs

```diff
@@ -0,0 +1,288 @@
+﻿using Microsoft.Extensions.Configuration;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using Diagnostics.RuntimeHost.Services.CacheService;
+using System.Net.Http.Headers;
+using Diagnostics.ModelsAndUtils.Models;
+using System.Data;
+using Microsoft.CodeAnalysis;
+using Diagnostics.ModelsAndUtils.Attributes;
+
+namespace Diagnostics.RuntimeHost.Services.DiagnosticsTranslator
+{
+    public interface IDiagnosticTranslatorService
+    {
+        Task<Response> GetResponseTranslations(Response response, string language);
+        Task<IEnumerable<DiagnosticApiResponse>> GetMetadataTranslations(IEnumerable<DiagnosticApiResponse> listDetectorsResponse, string language);
+    }
+
+    public class DiagnosticTranslatorService : IDiagnosticTranslatorService
+    {
+        private IConfiguration _config;
+
+        private static string _translatorSubscriptionKey;
+        private static readonly string translatorBaseURL = "https://api.cognitive.microsofttranslator.com/";
```

Comment:
I think this should go to App Setting file. (Thinking about other clouds as well)